content/e/ppw/2025-02-21-ppw-hardcore-friday.md

```diff
@@ -70,7 +71,7 @@ Hardcore Friday 21.000 was a show by [PpW Ewenement](@/o/ppw.md), held on Friday
 * This was the first PpW event to be streamed live at the [PpW VOD][ppw-vod], after the [previous show](@/e/ppw/2025-01-25-ppw-gruba-przesada.md) experienced technical difficulties and streamed on YouTube instead.
 * The live stream went offline a few times throughout the show, including during the final minutes of the opening match.
 * Mister Z defeated Marcelito in 2:47, thus setting a time limit for Biesiad to defeat Sambor.
-* The Vodka Match appeared to end in a double drunk-out, but Johnny Blade managed to get up to his feet, thus securing the win.
+* The Vodka Match ended in a no contest, since neither contestant managed to get up to his feet before the referee counted to ten. Johnny Blade appeared to raise just in time, but the [official results][hcf21k-wyniki] posted by PpW state there was no winner.
```

Comment:
```suggestion
* The Vodka Match ended in a no contest, since neither contestant managed to get up to his feet before the referee counted to ten. Johnny Blade appeared to get to his feet just in time, but the [official results][hcf21k-wyniki] posted by PpW state there was no winner.
```